openmoc/options.py

```diff
@@ -65,30 +75,69 @@ def __init__(self):
         self._num_omp_threads = multiprocessing.cpu_count()
         self._num_thread_blocks = 64
         self._num_gpu_threads = 64
+
+        self._opts = None
+        self._args = None
+
         self.parseArguments()
 
+    @property
+    def short_args(self):
+        return self._short_args
+
+    @property
+    def long_args(self):
+        return self._long_args
+
+    @property
+    def opts(self):
+        return self._opts
+
+    @property
+    def args(self):
+        return self._args
+
+    @property
+    def num_azim(self):
+        return self._num_azim
+
+    @property
+    def track_spacing(self):
+        return self._track_spacing
+
+    @property
+    def max_iters(self):
+        return self._max_iters
+
+    @property
+    def tolerance(self):
+        return self._tolerance
+
+    @property
+    def num_omp_threads(self):
+        return self._num_omp_threads
+
+    @property
+    def num_thread_blocks(self):
+        return self._num_thread_blocks
+
+    @property
+    def num_threads_per_block(self):
```

Comment:
All other attributes besides `_num_gpu_threads` use their name in the property method getter. What do you think about having this `def num_gpu_threads(self):`?